README.md

```diff
@@ -138,6 +138,29 @@ services:
 
 To start OpenMRS with your own database, just drop your data file (`.sql` or `.sql.gz`) in the [./sqls/mysql/](./sqls/mysql/) folder and recreate your volumes (`docker-compose -v down`).
 
+### Start with a custom PostgreSQL dump
```

Comment:
What about restoring the filestore piece?

Reply:
For the filetore we already have the `.env` variables to specify paths. should we include this in the README?

Reply:
I think we should explicitely document how to restore the filestores as well.